android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -454,6 +496,9 @@ public void loadURLObject(@Nullable Bundle urlObject) {
         // welcomePageEnabled
         props.putBoolean("welcomePageEnabled", welcomePageEnabled);
 
+        props.putBoolean("addPeopleEnabled", addPeopleEnabled);
+        props.putBoolean("dialOutEnabled", dialOutEnabled);
+
```

Comment:
@saghul, what I understand is that an SDK consumer has to do all of the following:
1. setAddPeopleEnabled(true),
2. Optionally, setDialOutEnabled, and
3. implement delegate/listener launchNativeInvite.

Then a tap/click on the InviteButton (and the whole invite flow that starts from that tap/click) is represented by a InviteSearchController.

So why didn't we just go with something like:
1. A single new method/property in JitsiMeetView called, for example, setInviteSearchProvider(InviteSearchProvider). If the SDK consumer gives us that instance, then we know we need to show the InviteButton in the UI i.e. that's the same as addPeopleEnabled(true). JitsiMeetView to InviteSearchProvider can have a 1-to-1 relationship for the sake of simplicity.
2. Technically, JitsiMeetView is not concerned with dialOutEnabled (as far as I understand) because it's only necessary for performQuery which is invoked on the native side anyway. So instead of lugging that around the JitsiMeetView on the native and JS sides, the SDK consumer could've provided it directly to performQuery as one more param (it already provides query, internally adds the uuid/scope, so what's one more param...).
3. Then we report a tap/click on InviteButton to the InviteSearchProvider. For example, InviteSearchProvider#launchNativeInvite(InviteSearchController).

Yeah, I know things can be implemented in multiple ways... Anyway, I'm chiming in far too late.